integrations/huggingface-transformers/recipes/bert-base-12layers_prune80.md

```diff
@@ -0,0 +1,97 @@
+<!--
+Copyright (c) 2021 - present / Neuralmagic, Inc. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+   http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing,
+software distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+---
+# General variables
```

Comment:
Apply similar edits from integrations/huggingface-transformers/recipes/bert-base-6layers_prune80.md to this entire file? Thanks.